Hsiao/HSIAO.py

```diff
@@ -1,266 +0,0 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
-
-import sncosmo
-import numpy as np
-import matplotlib.pyplot as plt
-import random
-import pandas as pd
-np.set_printoptions(precision=2)
-
-
-
-class Hsiao:
-    
-    """
-    Parameters:
-    --------
-    images: 'int'
-        Number of lensed supernova images 
-            
-    redshift: 'float'
-        Lensed supernova images redshift
-        Parameter of the Hsiao model
-        
-    amplitude: 'float'
-        Parameter of the Hsiao model
-        (Test 1e-4)
-        
-    datatype: 'string'
-        Define the data on the graph returned by the Graph function.
-        ('Flux', 'Total_Flux_Without_Noise', 'Noise', 'Total_Flux_With_Noise')
-        
-    Returns: 
-    --------
-    Data generation (flux, total flux, noise, total flux + noise) 
-        about supernova and lensed supernova. 
-    
-    """ 
-    
-#################################################
-    
-    def __init__(self, images, redshift, amplitude, datatype):
-        
-        self.t0 = 55000.
-        self.bands = ('ztfg', 'ztfr', 'ztfi')
-        self.pers = 5.0/100
-        self.nobs = [91, 90, 30]   #/!\ ds code nb_obs
-        self.dnobs = [1, 1, 4]
-        self.ddnobs = [0.8,0.8,1.2]
-        self.model = sncosmo.Model(source = 'hsiao')
-        
-        self.redshift = redshift
-        self.images = images
-        self.amplitude = amplitude
-        self.datatype = datatype
-        
-#################################################        
-        
-    def dT(self):
-        
-        """
-        
-        Return:
-        -------
-        dt: 'array'  shape(#images)
-            Absolute (unobservable) times
-            
-        """
-        
-        dt = np.zeros(self.images)
-        for i in range(len(dt)-1):
-            dt[i+1] = dt[i] + random.SystemRandom().uniform(5, 15)
-        
-        return dt
-    
-#################################################
-    
-    def MU(self):
```

Comment:
I think it's better if mu is provided by the user. It's true that you could have such methods generate mu values for mu for you whenever you want, but then you need to make sure that this value for mu is stored. So making it a class attribute would be a good idea.